hosts_template.yml

```diff
@@ -60,11 +61,18 @@ regionA:
       product: "edge"
     appliance_hostname: "anapaya-edge"
     # ebgp_multihop: 2 # if the bare metal host and edge appliance have IP's on different subnets, you will want to uncomment this.
+
+    
     # Directory in which secret values are stored on the controller
     secrets_dir: "{{ undef(hint='You must specify a path to the generated secrets, e.g., /home/ubuntu/secrets') }}"
```

Comment:
Can we make this optional so InterCloud/Cyberlink can use the appliance GUI to generate secrets? (Using the GUI is their usual process.)